src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -173,25 +173,21 @@ E. Click the *Tasks* tab, select an *Preview_Results* task and click on the *Pre
 image::execute.gif[100000,2000]
 
 [[_AutoML]]
+
 == AutoML
 
 The `auto-ml-optimization` bucket contains the Multi_Tuners_AutoML workflow that can be easily used to find the operating parameters for any system whose performance can be measured as a function of adjustable parameters.
+It is an estimator that minimizes the posterior expected value of a loss function.
 This bucket also comes with a set of workflows' examples that demonstrates how we can optimize mathematical functions, MLOS workflows and machine/deep learning algorithms from scripts using AutoML tuners.
 
-=== AutoML variants
-===== Multi_Tuners_AutoML_V1
-image::MLOS-AutoML-V1.png[align=center]
-===== Multi_Tuners_AutoML_V2
-image::MLOS-AutoML-V2.png[align=center]
-===== Multi_Tuners_AutoML_V3
-image::MLOS-AutoML-V3.png[align=center]
-=== Tuning Algorithms
+=== AutoML Variants
 
-===== Multi_Tuners_AutoML
+All Multi_Tuners_AutoML variants propose six algorithms for hyperparameters' optimization.  The choice of the sampling/search strategy depends strongly on the problem tackled.
+Three variants of AutoML are proposed. Each variant comes with specific pipelines and visualization tools as described in the subsections below.
```

Comment:
It would be better to give here the name of a brief description of the three variants and after talk about them in subsections I think.